Library/Formula/idris.rb

```diff
@@ -19,8 +18,18 @@ class Idris < Formula
   depends_on "cabal-install" => :build
   depends_on "gmp"
 
+  depends_on "libffi" => :recommended
+  depends_on "pkg-config" => :build if build.with? "libffi"
+
   def install
-    install_cabal_package
```

Comment:
Looks good! I managed to mess up the merge though to get this change in... better commit incoming